dev-docs/RFCs/v4.0.0/names_for_concepts_types_functions.md

```diff
@@ -61,6 +61,10 @@ The following technical terms should be used in the documentation, the H3 codeba
     - each base cell has a "base cell number" (0--121), which is encoded into the `H3Index` representation of every H3 cell
     - there is a one-to-one correspondence between the "base cell number" and the `H3Index` representation of resolution `0` cells
         + e.g., base cell 0 has `H3Index` hexadecimal representation `'8001fffffffffff'`
+- **boundary**:
+    - all or part of the geometric points that enclose an H3 cell
+    - may include more than 6 points in the case that a cell is not a geometric hexagon
```

Comment:
`such as when a hexagon crosses an icosahedron boundary`?

Reply:
I like it! Added.